### DISCUSSION TITLE
New install of intellicenter

### DISCUSSION THREAD

**Discussion by larryboy44** [jlvaillant/intellicenter#72] [General]
Hello.  I just installed your component and noticed something so this is really just an fyi.  I'm in Canada, and depending on the subject our temps or either in F or C.   Outside temps are in C but pool temps are in F.  I know, a mess.
My HA install is in Metric and thus shows C.   My pool is in Imperial and shows in F.  So, when I show the value of the air or water temp entity it shows the correct value but puts a C behind it.   So right now, in winter, my pool temp entity is reporting 40 C instead of 40 F.

There is no expectation of a fix for this configuration just wanted you to know it's a thing.
Thanks for putting this together.

**Comment by dwradcliffe** [comment]
I think we need to see if we can read the unit from the device instead of HA.
